arch/arm64/Kconfig

```diff
@@ -146,6 +146,15 @@ config ARCH_HAVE_EL3
 		runing at EL3 is not necessary and system register for EL3
 		is not accessible
 
+config ARCH_BOOT_EL3
```

Comment:
can we use a more general config name, since we can implement not only bootloader but monitor with nuttx? BTW, it's better to change the type from bool to int with range 1-3, so we can implement hypervisor with nuttx in the future.

Reply:
What could be a better, more general config name?

Reply:
remove BOOT, e.g. something ARCH_ARM64_RUNLEVEL?

Reply:
Will change to ARCH_ARM64_EXCEPTION_LEVEL, maybe more descriptive

Reply:
Put it in a separate patch so that it's easier to verify changes.